x-pack/plugins/cases/common/api/connectors/get_connectors.ts

```diff
@@ -32,3 +36,4 @@ export const GetCaseConnectorsResponseRt = rt.record(
 );
 
 export type GetCaseConnectorsResponse = rt.TypeOf<typeof GetCaseConnectorsResponseRt>;
```

Comment:
This type also changed and I noticed it is being used in [x-pack/plugins/cases/server/client/user_actions/connectors.ts:297](https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/server/client/user_actions/connectors.ts#L272).

This function returns

```
push: {
    needsToBePushed,
    hasBeenPushed: hasBeenPushed(enrichedPushInfo),
    externalService: enrichedPushInfo?.externalService,
    latestUserActionPushDate: enrichedPushInfo?.latestPushDate.toISOString(),
    oldestUserActionPushDate: enrichedPushInfo?.oldestPushDate.toISOString(),
},
```

but shouldn't it now be

```
push: {
    needsToBePushed,
    hasBeenPushed: hasBeenPushed(enrichedPushInfo),
    details: {
        externalService: enrichedPushInfo?.externalService,
        latestUserActionPushDate: enrichedPushInfo?.latestPushDate.toISOString(),
        oldestUserActionPushDate: enrichedPushInfo?.oldestPushDate.toISOString(),
    },
},
```
?